src/utils/dns_utils.cpp

```diff
@@ -54,13 +70,15 @@ namespace
  * the unbound library.
  */
 
+#ifdef HAVE_UNBOUND
```

Comment:
what does HAVE_UNBOUND relate to in this context?

Reply:
if the library unbound is present we will use it to resolve the dns entry instead of dns over https